master/docs/developer/rtype-build.rst

```diff
@@ -14,10 +14,7 @@ Builds
     :attr timestamp complete_at: time at which this build was complete, or None if it's still running
     :attr integer results: the results of the build (see :ref:`Build-Result-Codes`), or None if not complete
     :attr unicode state_string: a string giving detail on the state of the build.
-
-    .. todo::
-
-        * Currently build properties aren't available in this resource type
+    :attr dict properties: a dictionary of properties attached to build.
```

Comment:
XX: this property dict is only filled out if the properties filterspec is set (with example on how to set it within data api and rest api, as this is a little bit tricky)